index.rst

```diff
@@ -14,12 +14,12 @@ taken from the `Next-Gen Sequence Analysis Workshops
 <http://software-carpentry.org/>`__ and `Data Carpentry
 <http://datacarpentry.org>`__.
 
-* All workshops are on UC Davis campus unless otherwise specified.
+* All workshops and community events are on UC Davis campus unless otherwise specified.
 * Workshops are open to everyone whether or not they are part
   of the UC Davis community, subject to space constraints.
 * Workshops are free unless otherwise specified.
 
-If you want to be notified of new workshops you can sign up for the
+If you want to be notified of new workshops or events happening in the data science community at UC Davis, you can sign up for the
```

Comment:
I'm a little concerned about confusion/overlap with the DSI community. Maybe just "events" here?